### ISSUE TITLE
"Pods memory usage" graph shows double the actual value

### ISSUE THREAD

**Issue by jls-tschanzc** [rancher/rancher#26678]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
1. Enable "Cluster Monitoring"
2. Wait for data to accumulate
3. Open the Grafana Dashboard
4. Go to "Cluster -> Node XXX -> Pods memory usage"
5. Check the memory usage of pod Y
6. Go to "Pods -> Select Namespace & Pod Y"
7. Check the "Memory Usage" (you can also check the graphs directly on the workload in rancher)

**Result:**
The "Pods memory usage" graph shows 2x the actual current RAM usage of the pod.
The Node total memory usage is correct.

**Other details that may be helpful:**
![Screenshot 2020-04-17 at 14 44 31](https://user-images.githubusercontent.com/42810252/79572275-f3c1e900-80bc-11ea-90cb-dc14499311fe.jpg)
![Screenshot 2020-04-17 at 14 44 56](https://user-images.githubusercontent.com/42810252/79572279-f4f31600-80bc-11ea-949e-066eddaef01a.jpg)
![Screenshot 2020-04-17 at 14 45 34](https://user-images.githubusercontent.com/42810252/79572282-f6244300-80bc-11ea-8fbb-42b1e182345c.jpg)

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.3.5
- Installation option (single install/HA): HA
- Monitoring: 0.0.7

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): AWS EKS
- Machine type (cloud/VM/metal) and specifications (CPU/memory): Managed EC2 VMs
- Kubernetes version (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.0", GitCommit:"9e991415386e4cf155a24b1da15becaa390438d8", GitTreeState:"clean", BuildDate:"2020-03-26T06:17:09Z", GoVersion:"go1.14", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.9-eks-502bfb", GitCommit:"502bfb383169b124d87848f89e17a04b9fc1f6f0", GitTreeState:"clean", BuildDate:"2020-02-07T01:31:02Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by sangeethah** [comment]
This issue is being tracked in https://github.com/rancher/rancher/issues/26594

**Comment by jiaqiluo** [comment]
Hi @jls-tschanzc thanks for reporting the bug. This fixed will be v2.3.7 release.
